modules/ossm-deploy-application-workloads-in-each-mesh.adoc

```diff
@@ -0,0 +1,192 @@
+   -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/httpbin/httpbin.yaml
+----
++
+[NOTE]
+====
+You can use the following command to wait for deployment to be ready:
+
+[source,terminal]
+----
+$ oc wait deployments -n app-ns-2 --for condition=Available
+----
+====
+
+.Verification
+
+After deploying the applications, use the `istioctl ps` command to verify that each application workload is managed by its assigned control plane.
+
+.. Run the following procedure to verify workloads assigned to the control plane in `istio-system-1`:
+[source,terminal]
+----
+$ istioctl ps -i istio-system-1
+----
+You should see output similar to the following example:
+
+.Example output
+[source,terminal]
+----
+NAME                                  CLUSTER        CDS              LDS              EDS              RDS              ECDS        ISTIOD                            VERSION
+httpbin-7f56dc944b-vwfm5.app-ns-1     Kubernetes     SYNCED (11m)     SYNCED (11m)     SYNCED (11m)     SYNCED (11m)     IGNORED     istiod-mesh-1-b69646b6f-kxrwk     1.23.0
+sleep-5577c64d7c-d675f.app-ns-1       Kubernetes     SYNCED (11m)     SYNCED (11m)     SYNCED (11m)     SYNCED (11m)     IGNORED     istiod-mesh-1-b69646b6f-kxrwk     1.23.0
+----
+
+.. Run the following procedure to verify the workloads assigned to the control plane in `istio-system-2`:
+[source,terminal]
+----
+$ istioctl ps -i istio-system-2
+----
+You should see output similar to the following example:
+
+.Example output
+[source,terminal]
+----
+NAME                                  CLUSTER        CDS                LDS                EDS                RDS                ECDS        ISTIOD                            VERSION
+httpbin-7f56dc944b-54gjs.app-ns-3     Kubernetes     SYNCED (3m59s)     SYNCED (3m59s)     SYNCED (3m59s)     SYNCED (3m59s)     IGNORED     istiod-mesh-2-8666fdfc6-mqp45     1.23.0
+httpbin-7f56dc944b-gnh72.app-ns-2     Kubernetes     SYNCED (4m1s)      SYNCED (4m1s)      SYNCED (3m59s)     SYNCED (4m1s)      IGNORED     istiod-mesh-2-8666fdfc6-mqp45     1.23.0
+sleep-5577c64d7c-k9mxz.app-ns-2       Kubernetes     SYNCED (4m1s)      SYNCED (4m1s)      SYNCED (3m59s)     SYNCED (4m1s)      IGNORED     istiod-mesh-2-8666fdfc6-mqp45     1.23.0
+sleep-5577c64d7c-m9hvm.app-ns-3       Kubernetes     SYNCED (4m1s)      SYNCED (4m1s)      SYNCED (3m59s)     SYNCED (4m1s)      IGNORED     istiod-mesh-2-8666fdfc6-mqp45     1.23.0
+----
+
+Use the following procedure to verify that application connectivity is restricted to workloads within their respective mesh:
+
+. Send a request from the `sleep` pod in `app-ns-1` to the `httpbin` service in `app-ns-2`. The communication should fail:
++
+[source,terminal]
+----
+$ oc -n app-ns-1 exec deploy/sleep -c sleep -- curl -sIL http://httpbin.app-ns-2.svc.cluster.local:8000
+----
++
+You should see output similar to the following example:
```

Comment:
```suggestion
The `PeerAuthentication` resources we created earlier enforce mutual TLS (mTLS) traffic with the `STRICT` mode in each mesh. As each mesh has its own root certificate, managed by the `istio-ca-root-cert` config map, they are not able to communicate with each other. Thus, you should see output similar to the following example indicating that communication failed. :
``` @luksa Maybe have a quick review of the above sentence to see if it is correct or could be improved. I thought it was important to provide a bit more explanation on why this example fails.

Reply:
SGTM